tools/setup.py

```diff
@@ -52,11 +52,12 @@ def setup_rpc(self):
             devnull = open(os.devnull, 'wb')
             subprocess.check_call("git checkout master", stdout=devnull, stderr=subprocess.STDOUT)
             subprocess.check_call("git fetch origin", stdout=devnull, stderr=subprocess.STDOUT)
-            subprocess.check_call("git reset --hard origin/master", stdout=devnull, stderr=subprocess.STDOUT)
+            subprocess.check_call("git reset --hard origin/master",
```

Comment:
Does this compile? Generally when there's a line break you would have a `\` at the end of the line. 
`subprocess.check_call("git reset --hard origin/master", \`

Reply:
u only need that for strings, see the [pep8 rules](https://www.python.org/dev/peps/pep-0008/#indentation)